octane-eclipse-plugin/src/com/hpe/octane/ideplugins/eclipse/ui/entitydetail/field/DescriptionComposite.java

```diff
@@ -56,7 +64,44 @@ public DescriptionComposite(Composite parent, int style) {
         stackLayoutComposite.showControl(loadingComposite);
 
         browserDescHtml = factory.createBrowser(stackLayoutComposite, SWT.NONE);
-        browserDescHtml.addLocationListener(new LinkInterceptListener());
+        browserDescHtml.addLocationListener(new LocationAdapter() {
```

Comment:
Is this the same LocationAdapter as the one in the comments? If so, can you please put it in a class like the old LinkInterceptListener, and use it in both places to avoid code duplication

Reply:
yep and yep